tests/test_late_interaction_embeddings.py

```diff
@@ -103,6 +112,42 @@
             [-0.03473, 0.04792, -0.07033, 0.02196, -0.05314],
         ]
     ),
+    "jinaai/jina-colbert-v2": np.array(
+        [
+            [0.0475, 0.0250, -0.2225, -0.1087, -0.0297],
+            [0.0211, -0.0844, -0.0070, -0.1715, 0.0154],
+            [-0.0062, -0.0958, -0.0142, -0.1283, -0.0218],
+            [0.0490, -0.0500, -0.1613, 0.0193, 0.0280],
+            [0.0477, 0.0250, -0.2279, -0.1128, -0.0289],
+            [0.0597, -0.0676, -0.0955, -0.0756, 0.0234],
+            [0.0592, -0.0858, -0.0621, -0.1088, 0.0148],
+            [0.0870, -0.0715, -0.0769, -0.1414, 0.0365],
+            [0.1015, -0.0552, -0.0667, -0.1637, 0.0492],
+            [0.1135, -0.0469, -0.0573, -0.1702, 0.0535],
+            [0.1226, -0.0430, -0.0508, -0.1729, 0.0553],
+            [0.1287, -0.0387, -0.0425, -0.1757, 0.0567],
+            [0.1360, -0.0337, -0.0327, -0.1790, 0.0570],
+            [0.1434, -0.0267, -0.0242, -0.1831, 0.0569],
+            [0.1528, -0.0091, -0.0184, -0.1881, 0.0570],
+            [0.1547, 0.0185, -0.0231, -0.1803, 0.0538],
+            [0.1396, 0.0533, -0.0349, -0.1637, 0.0429],
+            [0.1074, 0.0851, -0.0418, -0.1461, 0.0231],
+            [0.0719, 0.1061, -0.0440, -0.1291, -0.0003],
+            [0.0456, 0.1146, -0.0457, -0.1118, -0.0192],
+            [0.0347, 0.1132, -0.0493, -0.0955, -0.0341],
+            [0.0357, 0.1074, -0.0491, -0.0821, -0.0449],
+            [0.0421, 0.1036, -0.0461, -0.0763, -0.0488],
+            [0.0479, 0.1019, -0.0434, -0.0721, -0.0483],
+            [0.0470, 0.0988, -0.0423, -0.0654, -0.0440],
+            [0.0439, 0.0947, -0.0418, -0.0591, -0.0349],
+            [0.0397, 0.0898, -0.0415, -0.0555, -0.0206],
+            [0.0434, 0.0815, -0.0411, -0.0543, 0.0057],
+            [0.0512, 0.0629, -0.0442, -0.0547, 0.0378],
+            [0.0584, 0.0483, -0.0528, -0.0607, 0.0568],
+            [0.0568, 0.0456, -0.0674, -0.0699, 0.0768],
+            [0.0205, -0.0859, -0.0385, -0.1231, -0.0331],
```

Comment:
how were these values computed?

I ran this code

```
from colbert.infra import ColBERTConfig
from colbert.modeling.checkpoint import Checkpoint

ckpt = Checkpoint("jinaai/jina-colbert-v2", colbert_config=ColBERTConfig())
docs = ["Hello World"]
query_vectors = ckpt.queryFromText(docs)
doc_vectors = ckpt.docFromText(docs)

print([[round(x, 4) for x in y] for y in query_vectors[0, :, :5].numpy()])
print([[round(x, 4) for x in y] for y in doc_vectors[0, :5, :5].numpy()])
```

And that's what I've got

```
[
    [0.0477, 0.0255, -0.2224, -0.1085, -0.03],
    [0.0206, -0.0845, -0.0075, -0.1712, 0.0156],
    [-0.0056, -0.0957, -0.0147, -0.1277, -0.0225],
    [0.0486, -0.0499, -0.1609, 0.0194, 0.0274],
    [0.0481, 0.0253, -0.2278, -0.1126, -0.0294],
    [0.0599, -0.0678, -0.0956, -0.0757, 0.0236],
    [0.0592, -0.0862, -0.0621, -0.1084, 0.0155],
    [0.0874, -0.0714, -0.0772, -0.1414, 0.037],
    [0.1009, -0.0552, -0.0669, -0.163, 0.0493],
    [0.1135, -0.047, -0.0576, -0.1699, 0.0538],
    [0.1228, -0.0428, -0.0507, -0.1725, 0.0562],
    [0.1291, -0.0388, -0.042, -0.1753, 0.0569],
    [0.1365, -0.0337, -0.0326, -0.1786, 0.0574],
    [0.1439, -0.026, -0.024, -0.1831, 0.0574],
    [0.1527, -0.0099, -0.0179, -0.1874, 0.057],
    [0.1555, 0.0186, -0.023, -0.1801, 0.0539],
    [0.1389, 0.054, -0.0345, -0.1636, 0.0429],
    [0.1058, 0.0862, -0.0418, -0.1455, 0.0222],
    [0.0713, 0.1061, -0.0438, -0.1288, 0.0002],
    [0.0453, 0.1143, -0.0457, -0.1119, -0.019],
    [0.0346, 0.1131, -0.0487, -0.0952, -0.0338],
    [0.0355, 0.1073, -0.0493, -0.0823, -0.0438],
    [0.0424, 0.1041, -0.0459, -0.0761, -0.048],
    [0.048, 0.102, -0.0421, -0.0718, -0.0477],
    [0.0474, 0.0989, -0.0413, -0.0654, -0.0431],
    [0.0434, 0.095, -0.0415, -0.0589, -0.0345],
    [0.0408, 0.0897, -0.0405, -0.0554, -0.0197],
    [0.0433, 0.0811, -0.0407, -0.0545, 0.0056],
    [0.0514, 0.0629, -0.0446, -0.0549, 0.0368],
    [0.058, 0.048, -0.0527, -0.0607, 0.0568],
    [0.0561, 0.0447, -0.0661, -0.0702, 0.0764],
    [0.0204, -0.0856, -0.0386, -0.1232, -0.0332],
]
```